src/subcommands/projects.ts

```diff
@@ -154,6 +169,18 @@ async function showProject(args: Args): Promise<void> {
       `Repository:\thttps://github.com/${project.git.repository.owner}/${project.git.repository.name}`,
     );
   }
+  if (databases.length > 0) {
+    console.log(
+      `Databases:\t${
+        databases.map((db) => `[${db.branch}] ${db.databaseId}`).join(`\n\t\t`)
+      }`,
+    );
+  }
+  if (crons && crons.length > 0) {
+    console.log(
+      `Crons:\t\t${crons?.map(renderCron).join("\n\t\t")}`,
+    );
+  }
```

Comment:
Just to clarify, in case `crons` is null, this results in printing `Crons:    undefined`, and this is expected, right?

Reply:
the `If (crons &&...` guards against crons being null

Reply:
oh then the question mark in `crons?.map` can be removed, right?